### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by nodejs-crowdin-zz**
Excluded /blog from translation
Updated project configuration to export only fully translated files. Related discussion with @XhmikosR : #3050

I believe that's not a good way, because we will never translate the website https://github.com/nodejs/nodejs.org/pull/3050#issuecomment-606853490, so it's kind of temporary solution, until I'll update build scripts.

This PR still contain some files that's not translated, but it's going from copy/pasted files between locales. Crowdin thoughts that this files is actual translation for original file, and that's why it's here. I think we can live with it for now.

@nodejs/website 
@nodejs/i18n 
@nodejs/i18n-api

### CODE REVIEW

File: locale/es/about/resources.md [hunk 1]
```diff
@@ -30,3 +29,4 @@ las [Pautas Visuales](/static/documents/foundation-visual-guidelines.pdf).
     <td><a href="/static/images/logos/nodejs-new-white.ai">Node.js invertido con menos color AI</a></td>
   </tr>
 </table>
+
```
Comment (alexandrtovmach) [inline locale/es/about/resources.md]:
```suggestion
```

File: locale/es/docs/guides/debugging-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+### [Built-in Debugger](https://nodejs.org/dist/latest-v6.x/docs/api/debugger.html)
+
+Start `node debug script_name.js` to start your script under the builtin command-line debugger. Your script starts in another Node.js process started with the `--debug-brk` option, and the initial Node.js process runs the `_debugger.js` script and connects to your target.
+
+### [node-inspector](https://github.com/node-inspector/node-inspector)
+
+Debug your Node.js app with Chrome DevTools by using an intermediary process which translates the Inspector Protocol used in Chromium to the V8 Debugger protocol used in Node.js.
+
+<!-- refs -->
+
```
Comment (alexandrtovmach) [inline locale/es/docs/guides/debugging-getting-started.md]:
```suggestion
```

File: locale/es/download/releases.md [hunk 3]
```diff
@@ -20,3 +20,4 @@ Las versiones desde la 1.x hasta la 3.x fueron llamadas "io.js" ya que fueron pa
 * [Todas las versiones](https://nodejs.org/dist/)
 
 </div>
+
```
Comment (alexandrtovmach) [inline locale/es/download/releases.md]:
```suggestion
```

File: locale/fr/about/resources.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  <tr>
+    <td class="bg-white"><a href="/static/images/logos/nodejs-new-black.ai"><img src="/static/images/logos/nodejs-new-black.svg" alt="Node.js on light background"></a></td>
+    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-white.ai"><img src="/static/images/logos/nodejs-new-white.svg" alt="Node.js on dark background"></a></td>
+  </tr>
+  <tr>
+    <td><a href="/static/images/logos/nodejs-new-black.ai">Node.js standard with less color AI</a></td>
+    <td><a href="/static/images/logos/nodejs-new-white.ai">Node.js reversed with less color AI</a></td>
+  </tr>
+</table>
+
```
Comment (alexandrtovmach) [inline locale/fr/about/resources.md]:
```suggestion
```

File: locale/fr/docs/guides/debugging-getting-started.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+### [Built-in Debugger](https://nodejs.org/dist/latest-v6.x/docs/api/debugger.html)
+
+Start `node debug script_name.js` to start your script under the builtin command-line debugger. Your script starts in another Node.js process started with the `--debug-brk` option, and the initial Node.js process runs the `_debugger.js` script and connects to your target.
+
+### [node-inspector](https://github.com/node-inspector/node-inspector)
+
+Debug your Node.js app with Chrome DevTools by using an intermediary process which translates the Inspector Protocol used in Chromium to the V8 Debugger protocol used in Node.js.
+
+<!-- refs -->
+
```
Comment (alexandrtovmach) [inline locale/fr/docs/guides/debugging-getting-started.md]:
```suggestion
```

File: locale/fr/download/releases.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+* [Node.js 10.x](https://nodejs.org/dist/latest-v10.x/)
+* [Node.js 8.x](https://nodejs.org/dist/latest-v8.x/)
+* [Node.js 6.x](https://nodejs.org/dist/latest-v6.x/)
+* [Node.js 4.x](https://nodejs.org/dist/latest-v4.x/)
+* [Node.js 0.12.x](https://nodejs.org/dist/latest-v0.12.x/)
+* [Node.js 0.10.x](https://nodejs.org/dist/latest-v0.10.x/)
+* [All versions](https://nodejs.org/dist/)
+
+</div>
+
```
Comment (alexandrtovmach) [inline locale/fr/download/releases.md]:
```suggestion
```

File: locale/ro/about/resources.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  <tr>
+    <td class="bg-white"><a href="/static/images/logos/nodejs-new-black.ai"><img src="/static/images/logos/nodejs-new-black.svg" alt="Node.js on light background"></a></td>
+    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-white.ai"><img src="/static/images/logos/nodejs-new-white.svg" alt="Node.js on dark background"></a></td>
+  </tr>
+  <tr>
+    <td><a href="/static/images/logos/nodejs-new-black.ai">Node.js standard with less color AI</a></td>
+    <td><a href="/static/images/logos/nodejs-new-white.ai">Node.js reversed with less color AI</a></td>
+  </tr>
+</table>
+
```
Comment (alexandrtovmach) [inline locale/ro/about/resources.md]:
```suggestion
```

File: locale/ro/docs/guides/debugging-getting-started.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+### [Built-in Debugger](https://nodejs.org/dist/latest-v6.x/docs/api/debugger.html)
+
+Start `node debug script_name.js` to start your script under the builtin command-line debugger. Your script starts in another Node.js process started with the `--debug-brk` option, and the initial Node.js process runs the `_debugger.js` script and connects to your target.
+
+### [node-inspector](https://github.com/node-inspector/node-inspector)
+
+Debug your Node.js app with Chrome DevTools by using an intermediary process which translates the Inspector Protocol used in Chromium to the V8 Debugger protocol used in Node.js.
+
+<!-- refs -->
+
```
Comment (alexandrtovmach) [inline locale/ro/docs/guides/debugging-getting-started.md]:
```suggestion
```

File: locale/ro/download/releases.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+* [Node.js 10.x](https://nodejs.org/dist/latest-v10.x/)
+* [Node.js 8.x](https://nodejs.org/dist/latest-v8.x/)
+* [Node.js 6.x](https://nodejs.org/dist/latest-v6.x/)
+* [Node.js 4.x](https://nodejs.org/dist/latest-v4.x/)
+* [Node.js 0.12.x](https://nodejs.org/dist/latest-v0.12.x/)
+* [Node.js 0.10.x](https://nodejs.org/dist/latest-v0.10.x/)
+* [All versions](https://nodejs.org/dist/)
+
+</div>
+
```
Comment (alexandrtovmach) [inline locale/ro/download/releases.md]:
```suggestion
```

**Review by alexandrtovmach — COMMENTED** [review]
formatting issue that happens only for already translated docs. Fixed once and shouldn't happen again

**Comment by alexandrtovmach** [comment]
Excluded `/blog` from translation
Updated project configuration to export only fully translated files. Related discussion with @XhmikosR : #3050 

I believe that's not good way, because we will never translate the website (https://github.com/nodejs/nodejs.org/pull/3050#issuecomment-606853490), so it's kind of temp solution, until I'll update build scripts.

This PR still contain some files that's not translated, but it's going from copy/pasted files between locales. Crowdin thoughts that this files is actual translation for original file, and that's why it's here. I think we can with it for now.

**Review by alexandrtovmach — APPROVED** [review]

**Comment by XhmikosR** [comment]
I still see English-only files in this branch...

**Comment by alexandrtovmach** [comment]
Please read description:
> This PR still contain some files that's not translated, but it's going from copy/pasted files between locales. Crowdin thoughts that this files is actual translation for original file, and that's why it's here. I think we can live with it for now.

**Comment by ghost** [comment]
Considering there are some files untranslated, I'll put it into "On hold" state until all of these files are translated, but if you think it's unnecessary, you can remove the label as well.

**Comment by alexandrtovmach** [comment]
@MaledongGit It's unnecessary because this PR is initial bunch of translations. I wouldn't like to wait until it become translated, because it can take long.

**Comment by alexandrtovmach** [comment]
I'm going to merge it, because it's blocking me to continue configuring localization

**Pull request merged**